### DISCUSSION TITLE
WDIO with GeoEgde running remote selenium grid

### DISCUSSION THREAD

**Discussion by paussita22** [webdriverio/webdriverio#13819] [Q&A]
Hi, 
I'm trying to run GeoEdge proxy on a test that is using webdriverio and is running remotely in Selenium Grid but I'm not able to get it working, I try to pass the values in the capabilities of the browser but didn't work, wonder if you have some idea of what I'm missing?

```
 proxy: {
    proxyType: 'manual',
    httpProxy: 'ip:port',
    socksUsername: 'username',
   socksPassword: 'pass',
    sslProxy: 'ip:port',
  },
```

**Comment by christian-bromann** [comment]
@paussita22 thanks for raising the issue, I am working on #13852 to re-enable proxies again in v9

**Reply by paussita22** [reply]
Thanks for working on this!
I'll keep an eye to test it with v9
